<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="timeLCD">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="tempLCD">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sgButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Singapore</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cnButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>390</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Canada</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ukButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>390</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>UK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jpButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>390</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Japan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="myButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>390</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Malaysia</string>
    </property>
   </widget>
   <widget class="QLabel" name="countryImage">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="countryImage_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="countryImage_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Temperature:</string>
    </property>
   </widget>
   <widget class="QLabel" name="countryImage_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>250</y>
      <width>51</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Â°C</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
